core/language/en-GB/Fields.multids

```diff
@@ -31,5 +31,6 @@ subtitle: The subtitle text for a wizard
 tags: A list of tags associated with a tiddler
 text: The body text of a tiddler
 title: The unique name of a tiddler
+toc-link: A value of <<.value no>> will supress the tiddler's link in a Table of Contents tree
```

Comment:
is toc-link part of empty.html?

Reply:
I think `<<.value no>>` is not part of the core and it shouldn't

Reply:
**toc-link** is part of: **$:/core/macros/toc**
It is  documented on tiddlywiki.com in tiddler: **Table-of-Contents Macros**
The Documentation tiddler showing this is on tiddlywiki.com : **TiddlerFields**

_<<.value no>>_ is part of the documentation macros in tiddler: **Documentation Macros**
This is not part of the core - but it is on tiddlywiki.com - for assisting in documentation.
Since the tiddler: **Documentation Macros** : is not on empty.html - there should be no conflict


I do not see any conflicts.

Reply:
we can't use `<<.value no>>` macro for core/language multids. ... If the macro doesn't exist it "eats" the text: no ... You may use `` `no` `` instead of the macro

Reply:
Thanks for spotting that conflict.
I will check the documentation style being used in other tiddlers before deciding which is more appropriate : `` `no` `` or  `''no''`